src/org/jitsi/impl/neomedia/rtp/translator/RTPTranslatorImpl.java

```diff
@@ -585,6 +593,11 @@ int didRead(
                 int pt = buf[off + 1] & 0x7f;
 
                 format = streamRTPManager.getFormat(pt);
+
+                // Pass the packet to the feedback message sender to update
+                // its transactions.
+                rtcpFeedbackMessageSender.maybeStopRequesting(
+                    streamRTPManager, ssrc, pt, buf, off, len);
```

Comment:
@gpolitis, "Pass the packet to the feedback message sender" is what the source code says so I find it stated in the comment unhelpful. I would like to read in the comment why it's necessary and, while "update its transactions" is in that spirit, I have no idea what transactions means. I guess I expect something more accessible. Anyway, that's my opinion and you don't have to act on it.